Sources/Factory/Factory/Containers.swift

```diff
@@ -44,7 +44,7 @@ import Foundation
 ///  See <doc:Containers> for more information.
 public final class Container: SharedContainer {
     /// Define the default shared container.
-    #if swift(>=6.1)
+    #if swift(>=5.5)
```

Comment:
Thanks for this. There was no reason for `@TaskLocal` to be tied to 6.1.

Reply:
Looks great. One nit is that I'm unsure about the examples for singleton since if you're doing manual testing you usually need to use withValue on the singleton AND the container.